### PR TITLE
Th css layout

### LINKED ISSUE #49: Choose visual color impairment color pallet
* Choose color pallet
* Implement color pallet

### PR DESCRIPTION
**Pull request by relyt4me**
### What’s this PR do?
Built new HTML and UI to display information more clearly,
Utitlized Sass mixins and variables as well as seperation of files

### Where should the reviewer start?
index.html and all scss files

### Any background context you want to provide?
This will break the connection between the js and html need to go in and repair post

### What are the relevant tickets?
resolves #49 resloves #26

### CODE REVIEW

File: src/css/_main-section.scss [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+      font-size: 2em;
+      padding-bottom: 5%;
+    }
+    p {
+      font-weight: bold;
+    }
+  }
+}
+.week-info-sec {
+  @extend .today-info-sec;
```
Comment (carlymclift) [inline src/css/_main-section.scss:30]: Great use of the the Mixins

File: src/css/_main-section.scss [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+.today-info-sec {
+  @include background-cards();
+  margin: 8px;
+  height: calc((100vh - 115px) * 0.35 - 9px);
+  h2 {
+    text-align: center;
+  }
+  article {
+    @include small-card();
```
Comment (carlymclift) [inline src/css/_main-section.scss:14]: Great use of mixins

File: src/css/_main-section.scss [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+//     font-size: 1em;
+//     margin: .5em;
+//     font-family: sans-serif;
+//     display: flex;
+//     flex-direction: column;
+//     justify-content: flex-start;
+//     width: 50%;
+//     height: 93%;
+//     overflow: scroll;
+//   }
```
Comment (carlymclift) [inline src/css/_main-section.scss:189]: Is all this commented out code not being used anymore?

Comment (relyt4me) [reply src/css/_main-section.scss:189]: Yes, I was keeping it as refrence for the older list if we needed to revert backwards

**Pull request merged**